### DISCUSSION TITLE
TOC with highlighted headings

### DISCUSSION THREAD

**Discussion by ToniMarokoko** [affiliatetheme/affiliatetheme#6] [General]
hello how can i automatically highlight headings of a toc 

so when im scrolling they will be displayed higlighted?

like on this page: https://affiliatetheme.io/webhosting/

**Comment by olaf-schmitz** [comment]
I guess you mean the widget in the sidebar? This is using the plugin https://de.wordpress.org/plugins/table-of-contents-plus/ and https://de.wordpress.org/plugins/q2w3-fixed-widget/ 
So this has nothing to do with the Affiliatetheme or any other Theme. Simply done with Plugins.

**Comment by ToniMarokoko** [comment]
Yeah I know. I personally use LuckWP - the only thing that is missing are the highlighted headings. Did you use a special script for that? So when i scroll down the page, that the displayed headings will be marked red for example. 

![highlighted](https://user-images.githubusercontent.com/26651966/230779828-3ef5b46c-7ea1-4c73-a25b-c8169e048988.png)

**Reply by olaf-schmitz** [reply]
this is simply done in CSS (CSS of the TOC Plug). Not sure LuckyWP offers this as separate CSS. TOC+ does.

**Comment by ToniMarokoko** [comment]
Ah okay. I thought this has to be some type of javascript. Can you provide me the css code for the TOC+ Plugin. I searched your site with developer tools but can't get it to work.

**Reply by olaf-schmitz** [reply]
I do not own the website, so have no clue on which CSS codes were used here.
